addon/components/listing.hbs

```diff
@@ -1,32 +1,98 @@
-<div class="au-o-grid">
-  <div class="au-o-grid__item au-u-4-5">
-    {{#each this.subForms as |subForm index|}}
-      <SubForm
-        @subForm={{subForm}}
-        @formStore={{@formStore}}
-        @graphs={{@graphs}}
-        @sourceNode={{subForm.sourceNode}}
-        @forceShowErrors={{@forceShowErrors}}
-        @cacheConditionals={{@cacheConditionals}}
-        @last={{eq index (sub this.subForms.length 1)}}
-        @show={{@show}}
-        @level={{@level}}
-        @onRemoveEntry={{this.removeEntry}}
-        @canRemove={{@listing.canRemove}}
-      />
-    {{/each}}
-  </div>
-  {{#if this.canAdd}}
-    <div class="au-o-grid__item au-u-1-5">
+{{#if @listing.isTable}}
```

Comment:
I've used big conditional blocks for now since that allowed me to iterate faster, but it might be worthwhile to work with different components instead?